pkg/providers/imagefamily/customscriptsbootstrap/provisionclientbootstrap.go

```diff
@@ -90,7 +91,7 @@ func (p ProvisionClientBootstrap) GetCustomDataAndCSE(ctx context.Context) (stri
 		OsType:                   lo.ToPtr(lo.Ternary(p.IsWindows, models.OSTypeWindows, models.OSTypeLinux)),
 		VMSize:                   lo.ToPtr(p.InstanceType.Name),
 		Distro:                   lo.ToPtr(p.ImageDistro),
-		CustomNodeLabels:         labels,
+		CustomNodeLabels:         nodeLabels,
```

Comment:
Dropping a comment just for tracking on what we covered offline, ensuring the ordering of kubelet identity used with `bootstrappingclient`, and `label` added will guaranteed always have the label at a <= version used.

Reply:
Maybe leave a comment somewhere about this?

Reply:
Did this